R/calcCapacityFactorHist.R

```diff
@@ -1,168 +1,120 @@
 #' @title calc Capacity Factor
 #' @description provides capacity factor values
 #'
-#' @param subtype data subtype. Either "wind" or "windoff"
+#' @param subtype data subtype. Either "windoff" or none
 #'
 #' @return magpie object of the capacity factor data
 #' @author Renato Rodrigues, Stephen Bi
 #' @examples
 #' \dontrun{
 #' calcOutput("CapacityFactor")
 #' }
-#'
-calcCapacityFactorHist <- function(subtype){
+#' 
+
 
-  if (subtype == "wind") {
```

Comment:
why do you need a subtype at all? it looks like this function is only called in fullREMIND and only with the default value for subtype

```
> madrat::getDependencies("calcCapacityFactorHist", direction = "dout")
```